app/assets/stylesheets/admin/orders.css.scss

```diff
@@ -91,3 +91,15 @@ th.actions {
 table.index td.actions {
   text-align: left;
 }
+
+.orders-loading {
```

Comment:
Should the spinner look the same everywhere? There are 11 other uses of the spinner. It would be great if we could have one partial and one block of CSS following BEM to use everywhere.

Reply:
Are we switching to BEM for all CSS now?

Reply:
We haven't discussed this properly, but several people like to use BEM. We don't have to switch everything. It can be a tool to make naming easier and I just thought this is a good example for BEM.

Reply:
Is it ok if I move that to a separate issue/PR? I can see 27 instances across 22 files that I would need to touch to make a single view and CSS for the spinner, and there's already quite a lot going on in this PR.

Reply:
Sure. A separate PR sounds great.